.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+    "presets": ["es2015"]
```

Comment:
Can you please explain what this preset does and which stage does it use. I would recommend using stage3 as the features made to stage3 will probably make into actual release.

Reply:
The es2015 preset is an older, official preset, it doesn't do very much in comparison to the stage-3 preset. Thanks for pointing that out, I changed over to the stage-3 preset.